infra/us-east-1/mgmt/atlantis/acm/terragrunt.hcl

```diff
@@ -0,0 +1,36 @@
+locals {
+  # Automatically load environment-level variables
+  environment_vars = read_terragrunt_config(find_in_parent_folders("env.hcl"))
+  account_vars     = read_terragrunt_config(find_in_parent_folders("account.hcl"))
+  # Extract out common variables for reuse
+  env         = local.environment_vars.locals.environment
+  region      = local.environment_vars.locals.aws_region
+  project     = local.environment_vars.locals.project
+  cidr_prefix = local.environment_vars.locals.cidr_prefix
+  tribe       = local.account_vars.locals.tribe
+  zone_id     = "Z10147571DRRDCJXSER5Y"
```

Comment:
You can get this from dependency outputs. We are managing that zone using Terraform.
See: https://github.com/input-output-hk/sc-dev-platform/blob/main/infra/global/route53/zones/terragrunt.hcl